src/alchemlyb/parsing/namd.py

```diff
@@ -9,14 +9,71 @@
 
 k_b = R_kJmol * kJ2kcal
 
+def get_lambdas(fep_files):
```

Comment:
Does this show up in the docs?

Build them locally with
```
python setup.py build_sphinx
```
and check.

(For some reason the RTD PR integration does not work #156.)

Reply:
Are users supposed to use `get_lambdas()` by themselves or would you consider this a "private" function of the module. If the latter, prefix with underscore and call it `_get_lambdas()` and then we also don't have to worry about documenting for users and we don't have to maintain the interface rigorously.

Reply:
get_lambdas() shouldn't be exposed to users. Our next PR will have the underscore prefix.